packages/cli/src/runAndroid/runAndroid.js

```diff
@@ -170,7 +170,11 @@ function tryInstallAppOnDevice(args, device) {
   try {
     // "app" is usually the default value for Android apps with only 1 app
     const appFolder = args.appFolder || 'app';
-    const pathToApk = `${appFolder}/build/outputs/apk/${appFolder}-debug.apk`;
+    const variant = (args.variant || 'debug').toLowerCase();
+    const buildDirectory = `${appFolder}/build/outputs/apk/${variant}`;
```

Comment:
I think it would be safer if you used `path.join` everywhere you used `/`, because it uses platform specific delimiter.

Reply:
Gradle respects posix paths only

Reply:
What about `fs` calls? It probably won't work on Windows. https://github.com/react-native-community/react-native-cli/blob/073389f5129a9b889a9db90b82e4278c4e847407/packages/cli/src/runAndroid/runAndroid.js#L199

Reply:
FS access needs to be normalized for Windows, yea. It's just paths that are injected into Gradle need to be posix (sorry for the confusion if that's not applicable here, I didn't review this PR yet :D )

Reply:
@Esemesek @thymikee Thanks for reviews. I used the [same method used in the existing source.](https://github.com/react-native-community/react-native-cli/blob/master/packages/cli/src/runAndroid/runAndroid.js#L93) I will try to change it in a better way.

Reply:
I don't think `fs` needs paths to be normalised, I think it does it by default. It's `path` that needs it.

Context: https://nodejs.org/api/path.html#path_windows_vs_posix

Reply:
@grabbou You are right. I've confirmed that it works fine on Windows. Thanks for the advice.